packages/core/src/application_ref.ts

```diff
@@ -239,6 +239,12 @@ export interface BootstrapOptions {
    * the change detection will only be trigged once.
    */
   ngZoneEventCoalescing?: boolean;
+
+  /**
+   * Optionally specify an array of ZoneSpecs, and those ZoneSpecs
+   * will be forked in NgZone instance.
```

Comment:
```suggestion
   * will be forked in the NgZone instance.
```
or maybe "an"? I think it needs an article, either way.

Reply:
Got it, thanks!